benchmarks/cases/project_linear.py

```diff
@@ -0,0 +1,27 @@
+import cartopy.io.shapereader as shpreader
+import cartopy.crs as ccrs
+import shapely.geometry as sgeom
+
+
+def time_ocean_project_pc():
+    shpfilename = shpreader.natural_earth(
+        resolution='50m', category='physical', name='ocean')
+    reader = shpreader.Reader(shpfilename)
+    oceans = reader.geometries()
+    pc = ccrs.PlateCarree()
+    oceans = sgeom.MultiPolygon(oceans)
+
+    # The thing we are really timing.
```

Comment:
I think you may want to look into [a setup function](https://asv.readthedocs.io/en/stable/writing_benchmarks.html#setup-and-teardown-functions) here.

Reply:
Done